index.rst

```diff
@@ -44,14 +44,579 @@
+
+.. note:: 
+
+   The deliverable description from the `charge`_ is in the note below to ease readability 
+
+   1. A summary of the suite of currently available tools/systems that may help fulfil the required functionality.
+
+      - Report on their status, limitations, and where their current requirements may need to be expanded to cover any increase in scope
+      - Tools which were considered but found to be inadequate must also be reported
+
+This section contains a summary of the tools currently available for use that are applicable to on-the-fly operations.
+What follows is a list of the tools, each with a short description that contain links to pertinent information.
+
+A high-level overview of these tools and computering resources was presented at the `original presentation given to stakeholders <https://docs.google.com/presentation/d/1i4p-sg42FXtEqGVqIZMeFadWSZZ0Lu_CpoqEafkMfy4/edit#slide=id.gd8dafc0d0d_0_30>`_.
+These slides may contain additional information and/or a more detailed description and use-case.
+
+Currently Available Tools
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The following tools are released for general use and are expected to be used in the context of on-the-fly information gathering and display.
+Some remain under active development whereas others are considered nearly complete.
+
+LOVE
+''''
+
+.. tiago writing this
+LOVE is a web-based user inteface for the observatory control system. 
+It is currently available at the `summit <http://amor01.cp.lsst.org/>` (VPN required) and all other test stands.
+
+The system is developed using the popular React JavaScript framework for the frontend, with Python in the backend.
+It contains an editing tool that allow users to create and customize views, but this feature rely exclusively on existing widget-like components.
+Extending the system beyond the vendor-provided "widgets" would require some knowledge of React and JavaScript, which is not common in the project.
+Nevertheless, LOVE does contain a especial widget that allows us to embed other web pages into a particular view. 
+This could be used, for instance, to embed `Bokeh Plotting Applications`_ alongside other LOVE components.
+
+
+
+Nublado (Jupyter Notebook) Interface
+''''''''''''''''''''''''''''''''''''
+
+.. patrick writing this
+The summit `Nublado interface <https://summit-lsp.lsst.codes/>`_ (VPN required) provides the user with a Jupyter Lab interface and the required libraries/packages to perform standard observatory operations including sending commands and running SAL scripts.
+It can also be used to query the EFD.
+This tool is setup to mimick the `RSP`_ and test-stand environments to the maximum extent possible, providing all the functionalities of a Jupyter Notebook but with direct access to the control system.
+Observers are expected to use this tool to perform commands or sequences that are not encapsulated into a SAL script.
+They are also useful for on-the-fly analysis and/or custom monitoring.
```

Comment:
"They are also useful" Not quite clear what "They" is referring to. "Jupyter Notebooks are also useful" or "The Nublado interface is also useful" ?